PowerShell Scanners/Windows Firewall Rules/Scan Profile.xml

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<AdminArsenal.Export Code="PDQInventory" Name="PDQ Inventory" Version="19.0.0.0" MinimumVersion="19.0">
+  <ScanProfile>
+    <Collections type="list" />
+    <Scanners type="list">
+      <Scanner>
+        <ModifiedDate>2020-04-02T12:12:46.0000000-06:00</ModifiedDate>
+        <Name>Windows Firewall Rules</Name>
+        <Script></Script>
+        <FileName>C:\PowerShell-Scanners\PowerShell Scanners\Windows Firewall Rules\Windows Firewall Rules.ps1</FileName>
+        <Parameters></Parameters>
+        <AdditionalFiles></AdditionalFiles>
+        <RowLimit value="100" />
```

Comment:
This needs to be higher. Even when I limit it to just enabled rules I have over 200.